.changeset/fast-wasps-turn.md

```diff
@@ -0,0 +1,5 @@
+---
+'@leafygreen-ui/button': patch
```

Comment:
This is also a breaking change, due to the CSS overflow change.

Reply:
👍